### ISSUE TITLE
When the maintainer wants to leave

### ISSUE THREAD

**Issue by JohnRoesler** [sous-chefs/website#68]
> ### When the maintainer wants to leave
> If someone objects to the project moving and wants to continue maintenance under the sous-chefs, we add a prominent link to fork on the readme and continue development. People are generally not idiots, so if they have a reason to leave us we should support them.
> Otherwise we update the cookbook README.md to mark as relocated, at least pointing where the cookbook went.

Seems to be talking more about when a maintainer wants to take a cookbook?

I would expect this section to talk about how to notify other sous-chef members that you want to no longer maintain and look for a new person

**Comment by damacus** [comment]
Yeah, we probably need two sections one for users leaving and one for cookbooks
